src/tests/ftest/util/ServerUtils.py

```diff
@@ -33,18 +33,126 @@
 import signal
 import fcntl
 import errno
+import yaml
 
 from avocado.utils import genio
 
 sessions = {}
 
+DEFAULT_YAML_FILE = "utils/config/examples/daos_server_sockets.yml"
+AVOCADO_YAML_FILE = "utils/config/daos_avocado_test.yml"
+
 class ServerFailed(Exception):
     """ Server didn't start/stop properly. """
 
 # a callback function used when there is cmd line I/O, not intended
 # to be used outside of this file
 def printFunc(thestring):
-        print("<SERVER>" + thestring)
+    print("<SERVER>" + thestring)
+
+def nvme_yaml_config(default_yaml_value, bdev, enabled=False):
+    """
+    Enable/Disable NVMe Mode.
+    By default it's Enabled in yaml file so disable to run with ofi+sockets.
```

Comment:
NVMe mode and ofi+sockets, might just not know something but these don't seem related to me?  Document parameters, I don't understand the function because I'm not sure what they are.

Reply:
In control plane API configuration file, utils/config/examples/*.yml. NVMe is enabled by default.
  # service with -n <path>)
  bdev_class: nvme
  bdev_list: ["0000:81:00.0"]  # generate regular nvme.conf

And we do not need this Enabled by default so this function will provide way to Enabled or Disabled that in yaml file

Reply:
Ah that comment is not correct. It's not related to ofi+sockets. will remove it from the comment.